### ISSUE TITLE
Add status bar version information display options

### ISSUE THREAD

**Issue by levifig** [tim-hub/obsidian-bible-reference#151]
Hey Tim,

I'd love to see a way to reduce the clutter this plugin adds to my status bar.

My suggestion is to give users a preference as to how it is (or not!) displayed:
- Hide
- Full _(e.g. "English Standard Version")_
- Short _(e.g. "ESV")_

Thank you for your time and work.

God bless.
–LF

**Comment by OfficerHalf** [comment]
Yep, this has led me to disable the plugin for the moment - it really sticks out especially with themes like Shimmering Focus that have a floating status bar.

**Comment by tim-hub** [comment]
Thanks for the feedback, was on a long holiday, will have a look

**Comment by tim-hub** [comment]
Thanks for the feedback, this has been added in in 2.3.7 release.
